src/App.js

```diff
@@ -4,17 +4,18 @@ import './App.css';
 import NavButton from './components/navbutton.js';
 import Shopping from './components/shopping.js';
 import AddItem from './components/addItem.js';
+import AddItemForm from './components/AddItemForm';
 
 function App() {
   return (
     <Router>
       <div className="App">
         <Route exact path="/" component={Shopping} />
         <Route path="/add" component={AddItem} />
-
+        <Route exact path="/add/form" component={AddItemForm} />
```

Comment:
🚨 Why make the path `/add/form`? Did you have another view in mind for `/add`? If not, I would keep `/add` for the add form.